dashboard/test/lib/services/user/password_resetter_test.rb

```diff
@@ -8,18 +8,32 @@ class Services::User::PasswordResetterTest < ActiveSupport::TestCase
   describe '#call' do
     subject(:reset_password) {described_instance.call}
 
-    let!(:user) {create(:user, email: email)}
     let(:mail) {ActionMailer::Base.deliveries.first}
 
-    it 'returns user' do
-      _reset_password.must_equal user
+    context 'for invalid email' do
```

Comment:
```suggestion
    context 'for invalid email' do
      let!(:user) {nil}
```
Usually when I see `context` there are `let`'s right after it which indicate the context being set.
I'm a little confused by this test because I can't tell what an "invalid email" is. Is for when `password_reset` is called but there is no current user?

Reply:
Yes, this test is for when an entered email doesn't match any existing user. I'll clarify the name of that test and add a proper context.